File: C/EX24.c
#include <ctype.h>
#include <stdio.h>
#include <math.h>


	void main()
    {
	    char sexo;
	    
	    printf("Digite o seu sexo M para masculino e F para feminino: ");
	    scanf("%s",&sexo);
	  
	    while((sexo != 'F') && (sexo != 'M'))
            {
	        printf("\nApenas F ou M sao respostas aceitas \nDigite novamente o seu sexo: ");
	        scanf("%s",&sexo);

	        }
	      if(sexo == 'M')
        {
	        printf("\nHomem\n");
	        
	    }
	    else if (sexo == 'F')
        {
	        printf("\nMulher\n");
	    }

	   return 0;
	}

File: Python/Ex24.py
sexo= str (input('Sexo F-feminino e M-masculino: '))

  
while(( sexo!= "F") and (sexo != "M")):

    sexo = str(input("\napenas F ou M são respostas aceitas \n Digite novamente o seu sexo: "))
    

if(sexo == "M"):

    print("\nHomem\n")
        
    
elif (sexo == "F"):

    print("\nMulher\n")